analyzer/src/funTest/kotlin/managers/MavenTest.kt

```diff
@@ -20,9 +20,9 @@
 package org.ossreviewtoolkit.analyzer.managers
 
 import org.ossreviewtoolkit.downloader.VersionControlSystem
-import org.ossreviewtoolkit.utils.getUserHomeDirectory
 import org.ossreviewtoolkit.utils.normalizeVcsUrl
 import org.ossreviewtoolkit.utils.safeDeleteRecursively
+import org.ossreviewtoolkit.utils.userHomeDirectory
```

Comment:
Commit message nit: "allow**s**"

Making them properties is not really required to achieve that, you could have added that call to the function as well. Isn't it more that you don't want to call the fixup function repeatedly?

Reply:
Correct, I've improved the commit message in that regard.